src/common/main.lua

```diff
@@ -1,29 +1,103 @@
 local json = require(".common.json")
 local utils = require(".common.utils")
 local main = {}
+-- just to ignore lint warnings
+local ao = ao or {}
 
-local camel = utils.camelCase
 main.init = function()
-	Owner = Owner or ao.env.Process.Owner
-	Name = Name or "ar.io ANT Registry"
+	ANT_REGISTRATION_TTL = 1000 * 60 * 30 -- 30 minutes
+	-- Example ANT structure
+	-- ANTS["antId"] = {
+	--     Owner = "userId",
+	--     Controllers = {"userId1" = true, "userId2" = true},
+	-- }
+	ANTS = ANTS or {}
```

Comment:
Since these are global, should we just hoist them to the file scope?

Reply:
agree, i don't think we need this function wrapper either